src/algebra/lie/basic.lean

```diff
@@ -333,15 +334,31 @@ instance has_coe_to_linear_equiv : has_coe (L₁ ≃ₗ⁅R⁆ L₂) (L₁ ≃
 /-- see Note [function coercion] -/
 instance : has_coe_to_fun (L₁ ≃ₗ⁅R⁆ L₂) (λ _, L₁ → L₂) := ⟨λ e, e.to_lie_hom.to_fun⟩
 
-@[simp, norm_cast] lemma coe_to_lie_equiv (e : L₁ ≃ₗ⁅R⁆ L₂) : ((e : L₁ →ₗ⁅R⁆ L₂) : L₁ → L₂) = e :=
+@[simp, norm_cast] lemma coe_to_lie_hom (e : L₁ ≃ₗ⁅R⁆ L₂) : ((e : L₁ →ₗ⁅R⁆ L₂) : L₁ → L₂) = e :=
 rfl
 
 @[simp, norm_cast] lemma coe_to_linear_equiv (e : L₁ ≃ₗ⁅R⁆ L₂) :
   ((e : L₁ ≃ₗ[R] L₂) : L₁ → L₂) = e := rfl
 
+@[simp] lemma to_linear_equiv_mk (f : L₁ →ₗ⁅R⁆ L₂) (g h₁ h₂) :
+  (mk f g h₁ h₂ : L₁ ≃ₗ[R] L₂) = { inv_fun := g, left_inv := h₁, right_inv := h₂, .. f } := rfl
+
+lemma coe_linear_equiv_injective : injective (coe : (L₁ ≃ₗ⁅R⁆ L₂) → (L₁ ≃ₗ[R] L₂)) :=
+begin
+  intros f₁ f₂ h, cases f₁, cases f₂, dsimp at h, simp only at h,
+  congr, exacts [lie_hom.coe_injective h.1, h.2]
+end
+
+@[ext] lemma ext (f g : L₁ ≃ₗ⁅R⁆ L₂) (h : (f : L₁ ≃ₗ[R] L₂) = g) : f = g :=
```

Comment:
This is pretty atypical for an ext lemma for a bundled equivalence. Can you add a brief comment explaining why we make a diff

Reply:
Also, it would work the same if you tagged `coe_linear_equiv_injective` with `ext`

Reply:
Probably I should just drop `ext` and use `ext'` (renamed to `ext`) as `@[ext]`.

Reply:
Done.